scripts/racmo_wind_vectors.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
```

Comment:
TODO: Move to `scripts/private_archive` or `scripts/private_archive_preprocess`?